osx/KBKit/KBKit/System/KBEnvConfig.m

```diff
@@ -36,26 +37,44 @@ + (instancetype)envConfigWithRunMode:(KBRunMode)runMode {
 - (instancetype)initWithRunMode:(KBRunMode)runMode {
   if ((self = [super init])) {
     _runMode = runMode;
+
+    // Read the mount point from the config file if possible.  NOTE:
+    // if you change this default, you must also change the default
+    // for darwin in the `GetMountDir` function of `libkb/env.go`.
+    NSString *defaultMountDir = [self homePath:[[self appNameWithDot] lowercaseString] options:0];
+    NSString *mountDir = defaultMountDir;
+    NSData *data = [NSData dataWithContentsOfFile:[self dataPath:@"config.json" options:0]];
+    if (data) {
+      NSError *err = nil;
+      NSDictionary *appConfig = [NSJSONSerialization JSONObjectWithData:data options:kNilOptions error:&err];
+      if (!err) {
+        id obj = [appConfig valueForKeyPath:@"mountdir"];
+        if (obj) {
+          mountDir = (NSString *)obj;
+        }
+      }
+    }
+
     switch (_runMode) {
       case KBRunModeProd: {
         self.title = @"Keybase.io";
-        self.mountDir = [KBPath path:@"/keybase" options:0];
+        self.mountDir = mountDir;
```

Comment:
This would be `~/keybase` now?

Reply:
Yep, assuming nothing is set in config.json.